### ISSUE TITLE
 Copy from VScode with color theme will cause an unexpect format

### ISSUE THREAD

**Issue by Su739** [facebook/draft-js#1720]
**feature**
Copy from VScode with color theme will cause an unexpect format, like this:

![record3](https://user-images.githubusercontent.com/31462528/38238739-6b530f0e-375d-11e8-94b7-6768767e9ea1.gif)
![record4](https://user-images.githubusercontent.com/31462528/38238740-6bad5572-375d-11e8-8094-4cc7a01cd79a.gif)

Am I going to ask VScode team?:)

**Comment by thibaudcolas** [comment]
Hey @Su739, it's not super clear for me looking at your examples what the problem is (GIFs are generally a bit hard to follow). Could you explain further what's the issue, and maybe provide a smaller and simpler example?

**Comment by Su739** [comment]
Tanks@thibaudcolas ,I mean I copy code from VScode IDE(and I use color theme in vscode) to draft-js Editor,then the code on the draft editor, will have a wrong space indentation in each row. I think the image is more intutive.
![draft](https://user-images.githubusercontent.com/31462528/38283676-e69443a4-37e9-11e8-8d9d-817c9f9990ca.png)
